### DISCUSSION TITLE
是否有办法禁用匿名评论

### DISCUSSION THREAD

**Discussion by tsq456** [walinejs/waline#179] [Q&A] [answered]
在文档里貌似没找到这个可配置的变量。

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/client/basic.html#anonymous

**Reply by free-nodes** [reply]
设置了 anonymous: false, 还是可以匿名评论

**Reply by lizheming** [reply]
@aiboboxx 是说直接调用接口评论吗？目前 anonymous 只是用于 UI 隐藏，没有真正去做接口这块的处理，这个后续可以优化一下。

**Reply by free-nodes** [reply]
看一下这里，
https://www.aiboboxx.ml/post/v2ray-mian-fei-dian-yue-di-zhi/
已经设置了 anonymous: false，还是可以匿名评论

**Reply by lizheming** [reply]
https://waline.js.org/client/basic.html#requiredfields 你把 requiredFields 设置一下吧，把 nick 设置成必填项试试

**Reply by free-nodes** [reply]
这样可以了，匿名不能提交评论，但是什么提示都没有，怎么弄个提示？

**Reply by tsq456** [reply]
> https://waline.js.org/client/basic.html#requiredfields 你把 requiredFields 设置一下吧，把 nick 设置成必填项试试

----
用的过程中发现个小问题，就不开新讨论了，看看后续是否有优化计划哈。

就是每次评论完，评论按钮变为禁用态之后，没有任何loading反馈，像valine评论加载时，在下方会有个loading图示。

而且不知道为什么waline生成评论的速度好像会比valine长些，我认为这样的情况下加载反馈就更重要了

**Reply by Mister-Hope** [reply]
> 这样可以了，匿名不能提交评论，但是什么提示都没有，怎么弄个提示？

建议在输入框自己提示，即设置placeholder自行引导。

**Reply by Mister-Hope** [reply]
> > https://waline.js.org/client/basic.html#requiredfields 你把 requiredFields 设置一下吧，把 nick 设置成必填项试试
> 
> ----
> 用的过程中发现个小问题，就不开新讨论了，看看后续是否有优化计划哈。
> 
> 就是每次评论完，评论按钮变为禁用态之后，没有任何loading反馈，像valine评论加载时，在下方会有个loading图示。
> 
> 而且不知道为什么waline生成评论的速度好像会比valine长些，我认为这样的情况下加载反馈就更重要了

加载反馈已添加
